test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
> Is there a good use case for `commute` with arrays?

Ramda is designed to work with other FL-compatible libraries (though it may be used alone). If this function were moved to ramda-fantasy, a Sanctuary user might need to add ramda-fantasy as a dependency just for this function. As a result we might end up defining `commute` in Sanctuary as well. Having `commute` in Ramda saves us from implementing it in N other libraries. :)